model/countries/FR/FR-parsing-rules.yaml

```diff
@@ -16,6 +16,12 @@ regex_definitions:
   # Regular expression to match the prefixes that indicate a house number or name.
   kHouseNumberOrNameOptionalPrefixRe:
     regex_fragment: '(?:(?:no|nr|°|º|numéro)[-.\s]*)?'
+  
+  kCommaOrNewlineSeparator:
```

Comment:
Isn't this similar to `kCommaOrWhitespaceSeparator`? I'm concerned about introducing multiple different separators on multiple country files. Ideally, we use shared ones as most as possible.

Reply:
French street names can have spaces in them, the kCommaOrWhitespaceSeparator ends up matching the first space and thus leading to the wrong parsing of the Place Charles de Gaulle

Reply:
I see. Then probably you want something like `(?:, |\n|\r|,)+`?